lldb/test/Shell/Swift/inlined-function-name-backtrace.test

```diff
@@ -0,0 +1,34 @@
+# Test Swift function name printing in backtraces.
+
+# XFAIL: !system-darwin
+# RUN: split-file %s %t
+# RUN: %target-swiftc -g -O %t/main.swift -o %t.out
+# RUN: %lldb -x -b -s %t/commands.input %t.out -o exit 2>&1 \
+# RUN:       | FileCheck %s
+
+#--- main.swift
+@inline(never)
+func baz(_ a: Int) -> Int {
+    return a * 2
+}
+
+@inline(__always)
+func foo(_ a: Int, _ b: Int) -> Int {
+ return a * b * baz(a) 
+}
+
+func bar() {
+    let baz = foo(4, 5)
+}
+
+bar()
+
+#--- commands.input
+b baz
+
+run
+bt
+
+# CHECK: `baz(a=Swift.Int @ scalar) at main.swift:3:14 [opt]
+# CHECK: `foo(a=Swift.Int @ scalar, b=Swift.Int @ scalar) at main.swift:8:17 [opt] [inlined]
```

Comment:
Give how suspicious the rendering of the parameters here looks, I would replace them with a `{{.*}}` since they're not part of what this test tries to test anyway.

Reply:
Fixed, and I removed the `XFAIL` on non Darwin since this should now pass on all platforms.